core/app/models/spree/product/scopes.rb

```diff
@@ -63,9 +63,10 @@ def self.property_conditions(property)
     #
     #   SELECT COUNT(*) ...
     add_search_scope :in_taxon do |taxon|
-      includes(:classifications).
-      where("spree_products_taxons.taxon_id" => taxon.self_and_descendants.pluck(:id)).
-      order(Spree::Classification.arel_table[:position].asc)
+      includes(:classifications)
+        .where('spree_products_taxons.taxon_id' => taxon.self_and_descendants.pluck(:id))
+        .select('spree_products_taxons.position')
+        .reorder(Spree::Classification.arel_table[:position].asc)
```

Comment:
why `reorder` instead of `order`?

Reply:
Feels natural considering we're using `order` in [here](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/taxon.rb#L10), but it works both ways.

Reply:
I would not change this if not strictly needed for this PR, as the `pluck` change above.

Reply:
Fair enough, let me fix it.